src/tune.cpp

```diff
@@ -18,4 +18,17 @@ TuneParameters& getTuneParameters() {
   return params;
 }
 
+std::ostream& operator<<(std::ostream& os, const TuneParameters& params) {
```

Comment:
Is `red2band_panel_nworkers` missing, right?

Reply:
Yep... fixed. Thanks!